From 463403b1c7eb8499dfe4c3bdf91255cc461db650 Mon Sep 17 00:00:00 2001
From: Marius Hoch <mail@mariushoch.de>
Date: Wed, 18 Sep 2024 12:07:00 +0200
Subject: [PATCH] hwdb: Add accel orientation quirk for the IdeaPad Duet 3
 10IGL5-LTE

Signed-off-by: Marius Hoch <mail@mariushoch.de>
(cherry picked from commit ff831e7c50350f92a86848b881fa320d528c1bd7)

Resolves: RHEL-49810
---
 hwdb.d/60-sensor.hwdb | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/hwdb.d/60-sensor.hwdb b/hwdb.d/60-sensor.hwdb
index 3e32c327e6..2adc5abc1f 100644
--- a/hwdb.d/60-sensor.hwdb
+++ b/hwdb.d/60-sensor.hwdb
@@ -760,8 +760,9 @@ sensor:modalias:i2c:bmc150_accel:dmi:*:svnLENOVO:*:pvrLenovoYoga300-11IBR:*
 sensor:modalias:acpi:ACCL0001*:dmi:*:svnLENOVO:pn60072:pvr851*:*
  ACCEL_MOUNT_MATRIX=0, 1, 0; -1, 0, 0; 0, 0, 1
 
-# IdeaPad Duet 3 10IGL5 (82AT)
+# IdeaPad Duet 3 10IGL5 (82AT) and 10IGL5-LTE (82HK)
 sensor:modalias:acpi:SMO8B30*:dmi:*:svnLENOVO*:pn82AT:*
+sensor:modalias:acpi:SMO8B30*:dmi:*:svnLENOVO*:pn82HK:*
  ACCEL_MOUNT_MATRIX=0, 1, 0; -1, 0, 0; 0, 0, 1
 
 #########################################
